ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -0,0 +1,278 @@
+#######
+# PIECEWISE SYSTEM CONFIGURATION
+#######
+
+# Certain tasks may operate in a less secure (but more convenient) manner, eg.
+# enabling passwordless sudo or generating self-signed ssl certs, when testing
+# locally, in Vagrant. But not in production!
+#
+# Supported values: production, development
+env: development
+
+# What Linux distro is being used? Supported options include: debian8, rhel7
+#sys: debian8
+
+# The project name should be the name of your forked repository.
+# This var is referenced by a few other vars, eg. git_repo, hostname, site_fqdn.
+project_name: piecewise
+
+# Information about the fork and branch to deploy.
+# Specify any valid git remote (typically a github username)
+remote: https://github.com/critzo
+
+# Specify any ref (eg. branch, tag, SHA) to be deployed. This ref must be
+# pushed to the remote git_repo before it can be deployed.
+piecewise_commit: ansible-refactor-rhel7-2
+
+git_repo: "{{ remote }}/{{ project_name }}.git"
+
+# BigQuery Project Number to use
+bigquery_project_num: 581276032543
+
+# Your Postgres database name and user can be customized if necessary.  
+database_name: piecewise
+database_user: postgres
+database_user_group: postgres
+
+# This is the fully qualified domain name of your production server. Because
+# nginx checks this value against the URL being requested, it must be the same
+# as the server's DNS name. 
+site_fqdn: "97.107.132.199"
+
+# The variables below will be used to generate SSL certificates. When deploying to 
+# development hosts, the country, state and city are used by open SSL to generate a 
+# self-signed certificate. This is useful when you don't have an DNS name set up yet 
+# and want to deploy using the host's IP address, or to a server not accessible to 
+# the Internet. 
+#
+# When deploying to a production host. The site_contact and the site_fqdn variables are 
+# used by EFF's certbot tool to automate the request for valid SSL certificates from 
+# the LetsEncrypt CA.
+site_contact: critzo@opentechinstitute.org
```

Comment:
Do you anticipate this ever being run by users/admins who haven't read the docs well?  This and other defaults that are specific to a particular deployment may end up being used as-is, so I'd be careful.  Maybe you could consider making ansible fail when they haven't been set rather than providing defaults that aren't universal.